sklearn/tests/test_common.py

```diff
@@ -297,3 +304,186 @@ def test_search_cv(estimator, check, request):
 def test_check_n_features_in_after_fitting(estimator):
     _set_checking_parameters(estimator)
     check_n_features_in_after_fitting(estimator.__class__.__name__, estimator)
+
+
+class MinimalClassifier:
```

Comment:
should we move this into a new file, like `_test_minimal_requirements`?
I feel like this isn't "common" in the sense that it's not a check that's run on all estimators (which is how I understand this file, could be wrong)

Reply:
I moved the estimator in `_testing.py` (I was on the previous comment when doing it).
Do you think that we need a separate file for the estimator?